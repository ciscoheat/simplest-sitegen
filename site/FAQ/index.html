<!-- build:title -->DCI Q&amp;A<!-- /build:title -->
<!-- build:styles --><!-- /build:styles -->
<!-- build:content -->
<h1 class="text-center">DCI Frequently Asked Questions</h1>
<hr size="1" />
<p>The FAQ is located in the Wiki. <a href="/doku.php?id=FAQ">Follow this link to the FAQ</a>, 
  and use the browser back button to return here.</p>
<!-- /build:content -->