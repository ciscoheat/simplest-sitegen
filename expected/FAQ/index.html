<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>DCI Q&amp;A</title>
  <link rel="stylesheet" href="/CSS/chota.min.css?glxzhr">
  <link rel="stylesheet" href="/CSS/fulloo.css?1dpkm1x">
  
</head>
<body>
  <nav class="nav hide-pr" id="tabs">
    <div class="nav-center">
      <a href="/">Home</a>
      <a href="/Introduction/">Introduction</a>
      <a href="/FAQ/">FAQ</a>
      <a href="/Documents/">Documents</a>
      <a href="/Examples/">Examples</a>
      <a href="/Downloads/">Downloads</a>
      <a href="/Events/">Events</a>
      <a href="/oowiki.php">Wiki</a>
    </div>
  </nav>

  <main class="container">
    
<h1 class="text-center">DCI Frequently Asked Questions</h1>
<hr size="1" >
<p>The FAQ is located in the Wiki. <a href="/doku.php?id=FAQ">Follow this link to the FAQ</a>, 
  and use the browser back button to return here.</p>
<img src="data:image/jpeg;base64,/9j/4QC4RXhpZgAATU0AKgAAAAgACAESAAMAAAABAAEAAAEaAAUAAAABAAAAbgEbAAUAAAABAAAAdgEoAAMAAAABAAIAAAExAAIAAAAOAAAAfgEyAAIAAAAUAAAAjAE8AAIAAAAQAAAAoAITAAMAAAABAAEAAAAAAAACQAAAAAEAAAJAAAAAAQAAUXVpY2tUaW1lIDcuNgAyMDA5OjAyOjA5IDE2OjMwOjIyAE1hYyBPUyBYIDEwLj">

  </main>
  
  <script src="/js/highlight.min.js"></script>
  <script>
    // Navigation highlight
    const current = document.location.pathname
    document.querySelectorAll('#tabs a').forEach(a => {
      const tab = new URL(a.href).pathname

      if(tab == current) {
        a.classList.add('active')
      }
    })
  </script>    
</body>
</html>
